osi_common.proto

```diff
@@ -154,6 +154,9 @@ message BaseStationary
 
     /// The relative orientation of the landmark w.r.t its parent frame.
     optional Orientation3d orientation = 3;
+    
+    /// The two dimensional (flat) contour of the object. 
```

Comment:
@SoykaDaimler
> we define the contour in world and sensor coordinates, and not relative to the position/orientation.
If the contour is defined in world coordinates, why is there no Z-component in the contour? On what plane is the z-component projected? Or am I missing something. If z is simply set to 0 there may be problems with lanes with elevation. Or can I imagine it as a list of points on a plane, that is then translated and rotated.

If it is defined in sensor coordinates it should be relative to position / orientation right? 

Regarding the reference frame discussions I would leave it as is for this pull request.